### ISSUE TITLE
Running workspace diagnostics only on request

### ISSUE THREAD

**Issue by noearchimede** [artemave/workspace-diagnostics.nvim#12]
## Describe the problem

First of all, thank you for publishing this plugin!
Having workspace diagnostics is very userful for me for some smaller workspaces, but in my personal workflow I often find it unnecessary and potentially annoying in larger ones.
I was wondering if it would be possible to trigger the search for diagnostics in all workspace files only when the user triggers a specific mapping or command instead of having it happen automatically when the LSP is first attached to a buffer in the workspace.

Based on the blogpost where you explain how this plugin works I figured that this should be possible, but I'm not able to figure out how to do it in practice. For example, I don't know what to use in place of `client` in the `require("workspace-diagnostics").populate_workspace_diagnostics(client, bufnr)` function call if it was triggered outside of the `on_attach` callback provided by lspconfig.

## Describe the solution

The solution would be any way to populate the workspace diagnostics with an explicit command rather than automatically upon opening the first file.

**Comment by artemave** [comment]
Here's how you can get the list of clients, attached to the current buffer:

```lua
vim.lsp.buf_get_clients()
```

**Comment by artemave** [comment]
I updated the docs for your use case.

**Comment by noearchimede** [comment]
Thank you !!
